run/test/specs/disappearing_messages.spec.ts

```diff
@@ -275,6 +248,10 @@ async function disappearAfterSendNoteToSelf(platform: SupportedPlatformsType) {
     selector: DISAPPEARING_TIMES.ONE_DAY,
   });
   await device.disappearRadioButtonSelected(DISAPPEARING_TIMES.ONE_DAY);
+  time =
+    platform === "ios"
```

Comment:
would be good to just do device.iOS() or equivalent, but this can wait another PR